src/components/container/sticky-bar/sticky-bar.stories.ts

```diff
@@ -0,0 +1,194 @@
+    labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+    laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+    voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+  </p>
+  <sbb-button
+    variant="secondary"
+    ?negative=${color === 'midnight'}
+    style="margin-block-end: 0.75rem;"
+    >See more</sbb-button
+  >
+`;
+
+const Template = (): TemplateResult =>
+  html` <sbb-sticky-bar>
+    <sbb-button variant="secondary">Example</sbb-button>
+  </sbb-sticky-bar>`;
+
+const DefaultTemplate = ({ color, containerColor, ...args }: Args): TemplateResult => html`
+  <sbb-container ${sbbSpread(args)} color=${containerColor}>
+    ${containerContent('Example title', containerColor)}
+    ${containerContent('Another one', containerColor)}
+    ${containerContent('And another one', containerColor)}
+    ${containerContent('And a last one', containerColor)}
+
+    <sbb-sticky-bar color=${color !== 'unset' ? color : nothing}> ${actionGroup()} </sbb-sticky-bar>
+  </sbb-container>
+`;
+
+const ShortTemplate = ({ color, containerColor, ...args }: Args): TemplateResult => html`
+  <sbb-container ${sbbSpread(args)} color=${containerColor}>
+    ${containerContent('Example title', containerColor)}
+
+    <sbb-sticky-bar color=${color !== 'unset' ? color : nothing}> ${actionGroup()} </sbb-sticky-bar>
+  </sbb-container>
+`;
+
+const WithStickybarTemplate = ({ color, containerColor, ...args }: Args): TemplateResult => html`
+  <sbb-container ${sbbSpread(args)} color=${containerColor}>
+    ${containerContent('Example title', containerColor)}
+    ${containerContent('Another one', containerColor)}
+    ${containerContent('And another one', containerColor)}
+    ${containerContent('And a last one', containerColor)}
+
+    <sbb-sticky-bar color=${color !== 'unset' ? color : nothing}> ${actionGroup()} </sbb-sticky-bar>
+  </sbb-container>
+  <sbb-container color="milk">
+    <div style="padding-block: 4rem;">
+      ${containerContent('Content after first container', 'milk')}
+      ${containerContent('Another one', 'milk')}
+    </div>
+  </sbb-container>
+`;
+
+export const Standalone: StoryObj = {
+  render: Template,
+};
+
+export const Default: StoryObj = {
+  render: DefaultTemplate,
+  argTypes: defaultArgTypes,
+  args: defaultArgs,
+};
+
+export const Expanded: StoryObj = {
+  render: DefaultTemplate,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, expanded: true },
+};
+
+export const ShortContent: StoryObj = {
+  render: ShortTemplate,
+  argTypes: defaultArgTypes,
+  args: defaultArgs,
+};
+
+export const White: StoryObj = {
+  render: DefaultTemplate,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, color: color.options[1] },
+};
+
+export const Milk: StoryObj = {
+  render: DefaultTemplate,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, color: color.options[2] },
+};
+
+export const Midnight: StoryObj = {
```

Comment:
for chromatic it would be great if we have two stories with a background color set in the container. one with color on sticky and one without. So we would have all combinations.

To test the sticking state in chromatic, maybe we can in some stories enforce a fixed height, so it shows sticky?

Reply:
I see new stories but without fixed height, or is the sticky bar sticky in chromatic anyways?